tensorflow_datasets/text/pg19_test.py

```diff
@@ -0,0 +1,35 @@
+# coding=utf-8
+# Copyright 2020 The TensorFlow Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Lint as: python3
+"""Test for PG-19 dataset module."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import tensorflow_datasets.public_api as tfds
+from tensorflow_datasets.text import pg19
+
+class Pg19Test(tfds.testing.DatasetBuilderTestCase):
+  DATASET_CLASS = pg19.Pg19
+  SPLITS = {
+      "train": 28602,   # Number of fake train example
```

Comment:
You are generating the full dataset every time the test run, rather than generating only the fake data. The reason is that you're reading from `_DATA_DIR`

Reply:
Yes, you're right. 
But If I'm not reading data from `_DATA_DIR` then how am I supposed to read data from to pass tests?
Any suggestion? Thank you

Reply:
You can patched the `_DATA_DIR` to use the fake path. See `wiki40b_test` as an example.

I've applied those changes internally, so you don't have to do it.

Reply:
Ok